### PR TITLE
Reuse filters and results layout

### LINKED ISSUE #944: 👍👍 Reuse filters and results layout on learn and events pages

### PR DESCRIPTION
**Pull request by y4izus**
Fixes #944 

On this PR, the Learn, Advocates and Events pages share the same component for creating the layout of the aside filters and the results container

### CODE REVIEW

File: components/learn/TheLearningResourceList.vue [hunk 1]
```diff
@@ -33,19 +33,29 @@
...
-        <fieldset class="bx--col-lg-4 bx--col-md-2 the-learning-resources-list__filter-time">
-          <legend
-            class="
-                the-learning-resources-list__filter-time-label
-                the-learning-resources-list__medium-large-only
-              "
-          >
-            Time to spend learning
-          </legend>
+
```
Comment (eddybrando) [inline components/learn/TheLearningResourceList.vue:36]: We could probably get rid of this line to keep it consistent with other pages:

```suggestion
```

File: pages/events/index.vue [hunk 2]
```diff
@@ -52,31 +40,44 @@
...
+          >
+            <div class="event-page__empty-card-description">
+              {{ emptyCard.description }}
+            </div>
+          </AppCard>
+          <EventCard
+            v-for="event in filteredEvents"
+            v-else
+            :key="`${event.title}-${event.place}-${event.date}`"
+            class="results-item"
```
Comment (eddybrando) [inline pages/events/index.vue:71]: I find it quite tricky to figure out where this definition is without prior knowledge (like this PRs code review).

I'm think of another approach that could be easier to understand.

1. Maybe we could define `.results.item` in the component using the definition (**pages/events/index.vue** and **pages/learn.index.vue**).
2. Or maybe we could leave the definition in **components/ui/AppFiltersResultsLayout.vue** but rename it to `.app-filters-results-layout__results-item` to hint to where the definition is.

**Review by eddybrando — APPROVED** [review]
Thanks a lot for the great PR! 🙂 

Please just consider my 2 suggestions before merging.

File: pages/events/index.vue (hunk 2 above)
Comment (y4izus) [reply pages/events/index.vue:71]: I thought the same when I did it 👍 . I like the second option

**Pull request merged**